gcloud-java-examples/src/main/java/com/google/gcloud/examples/StorageExample.java

```diff
@@ -213,8 +214,12 @@ String parse(String... args) {
     public void run(Storage storage, String bucketName) {
       if (bucketName == null) {
         // list buckets
-        for (BucketInfo b : storage.list()) {
-          System.out.println(b);
+        Page<BucketInfo> bucketPage = storage.list();
```

Comment:
Nit, Fine as is, but I do wonder... even though `do ~ while` loops are much less common in Java as 
our API will never return `null` from the `list`, and hopefully other, operations shouldn't the suggested usage be:

```java
Page<BucketInfo> bucketPage = storage.list();
do {
  for (BucketInfo b : bucketPage.values()) {
    System.out.println(b);
  }
  bucketPage = bucketPage.nextPage();
} while (bucketPage != null);
```